sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -32,6 +33,13 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
         path: AppRoutes.InstitutionDashboard,
         name: InstitutionRoutesConst.INSTITUTION_DASHBOARD,
         component: InstitutionDashboard,
+        children: [
```

Comment:
I won't consider the manage location as a child of the dashboard. Do you think that maybe would be good to have it as a main route under institution?

Reply:
I put it there under the dashboard because side navigation bar in UX.

Reply:
Afte the conversation we are leaving as is waiting for more solid requirements about it.